app/views/proposals/show.html.haml

```diff
@@ -97,3 +97,23 @@
               %dt Requires Registration:
               %dd
                 = link_to "Yes (#{registered_text(@event)})", new_conference_conference_registration_path(@conference.short_title), class: 'btn btn-xs btn-danger', disabled: !@event.registration_possible?
+          - if concurrent_events(@event).present?
+            .col-md-12#concurrent-events
+              %h4 Happening at the same time:
+              %ul.nav.nav-pills.nav-stacked
+                - concurrent_events(@event).each do |event|
+                  %li
+                    = link_to conference_program_proposal_path(@conference.short_title, event.id) do
+                      %strong
+                        = event.title
+                  %dl
+                    .col-md-12
+                      %dt Start Time:
+                      %dd
+                        = event.program.selected_event_schedules.find_by(event: event).start_time.strftime("%Y %B %e %H:%M")
+                    .col-md-12
+                      %dt Room:
+                      %dd
+                        = event.room.name
+                  %br
```

Comment:
I don't think we really need any of the CSS you are adding (or the `.nav.nav-pills` for that matter). Why don't you try a simpler approach? Perhaps a horizontal line can separate the proposal info from the `Happening at the same time` info. I would also use numbering for showing the concurrent events.